sass/components/chart-mode.scss

```diff
@@ -59,7 +59,7 @@
 }
 
 .smartcharts-mobile {
-    .sc-chart-mode .cq-dialog {
+    .sc-chart-mode .sc-dialog {
```

Comment:
@shayan-binary-2 , please note that in deriv.app there are instances where we are overriding the classNames with `cq` prefix. Just to give a heads up again.

Reply:
@zaki-binary yes exactly. after releasing the chart, I am going to integrate it with deriv.app and update any places which we use these overriding